app/controllers/solicitations_controller.rb

```diff
@@ -13,6 +13,7 @@ def create
 
     @result = 'success'
     @partial = 'solicitations/thank_you'
+    CompanyMailer.confirmation_solicitation(@solicitation.email).deliver_now
```

Comment:
```suggestion
    CompanyMailer.confirmation_solicitation(@solicitation.email).deliver_later
```
huhuhu :)

Ou alors `CompanyMailer.delay.confirmation_solicitation…` mais j’ai fouillé un peu comment faire pour tester les jobs en développement sans lance `rake jobs:work` et on peut simplement spécifier la queue active_job à :inline. Je prépare une petite PR pour ça. Par contre, utiliser `.delay` ou `.deliver_later` ne revient pas exactement au même: `.delay` utilisera forcément un DelayedJob, tandis que `.deliver_later` respecte la queue configurée. Il faut donc plutôt écrire `.deliver_later`.

Reply:
PR #738 sur ce sujet :)